tests/fix_rights.c

```diff
@@ -11,6 +11,7 @@
 char *VENV_PATH = "/hepcrawl_venv/";
 char *CODE_PATH = "/code/";
 char *TMP_PATH = "/tmp/";
+char *VAR_PATH = "/var/";
```

Comment:
instead of this you should use a configuration override to set it to somewhere inside the test directory.